static/bidder-info/pulsepoint.yaml

```diff
@@ -4,6 +4,12 @@ capabilities:
   app:
     mediaTypes:
       - banner
+      - video
+      - audio
+      - native
```

Comment:
Could you please add a json test for a native ad?

Reply:
done